### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by LarsKumbier**
Adds the ability to change the swipe rotation for rotated components

In one of our projects, we used the react-image-gallery, but needed to rotate it for different views. Since the swipe component react-swipeable was unable to rotate the swipes accordingly, we added the appropriate parts in this PR. Changes are non-breaking.

### CODE REVIEW

File: src/Swipeable.js [hunk 1]
```diff
@@ -25,8 +25,15 @@ function getPosition(e) {
     : { x: e.clientX, y: e.clientY };
 }
 
+function rotateByAngle({ x, y }, angle) {
+  const angleInRadians = (Math.PI / 180) * angle;
```
Comment (hartzis) [inline src/Swipeable.js]: thoughts on quick out?
```
function rotateByAngle(positions, angle) {
  if (angle === 0) return positions;
```
and don't destructure, since that will be the majority, currently, of use cases?

File: README.md [hunk 2]
```diff
@@ -89,6 +89,8 @@ as well as the x distance, + or -, from where the swipe started to where it ende
 
 **`innerRef`** will allow access to the Swipeable's inner dom node element react ref. See [#81](https://github.com/dogfessional/react-swipeable/issues/81) for more details. Example usage `<Swipeable innerRef={(el) => this.swipeableEl = el} >`. Then you'll have access to the dom element that Swipeable uses internally.
 
+**`rotationAngle`** will allow to set a rotation angle, e.g. for a four-player game on a tablet, where each player has a 90° turned view. The default value is `0`.
```
Comment (hartzis) [inline README.md]: Should we "lock" this to a range of `0 - 360`? via a new proptype?

File: src/__tests__/Swipeable.spec.js [hunk 3]
```diff
@@ -379,4 +380,125 @@ describe('Swipeable', () => {
       expect(swipeableDiv.prop('onTouchMove')).toBe(instance.eventMove);
     });
   });
+
+  describe('Handle Rotation by 90 degree: ', () => {
```
Comment (hartzis) [inline src/__tests__/Swipeable.spec.js]: Thank you for adding tests!!!

File: README.md (hunk 2 above)
Comment (LarsKumbier) [reply README.md]: We thought about the same, but in the end it does not matter for the calculation - so why lock the range.

File: src/__tests__/Swipeable.spec.js (hunk 3 above)
Comment (LarsKumbier) [reply src/__tests__/Swipeable.spec.js]: 'course - won't work without. ;)

File: src/Swipeable.js (hunk 1 above)
Comment (LarsKumbier) [reply src/Swipeable.js]: will add that

File: src/__tests__/Swipeable.spec.js (hunk 3 above)
Comment (LarsKumbier) [reply src/__tests__/Swipeable.spec.js]: ah, a question: the tests are mainly data-driven. Is it okay to refactor them into data-driven tests with less duplication?

**Comment by LarsKumbier** [comment]
@hartzis Could you re-review? Added more tests for negative degrees and over-360-degrees for documentation and the early-return as requested.

File: src/__tests__/Swipeable.spec.js (hunk 3 above)
Comment (LarsKumbier) [reply src/__tests__/Swipeable.spec.js]: Yeah, externally
